database/mongodb/mongodb.go

```diff
@@ -36,8 +36,9 @@ const LockIndexName = "lock_unique_key"                  // the name of the inde
 const contextWaitTimeout = 5 * time.Second               // how long to wait for the request to mongo to block/wait for.
 
 var (
-	ErrNoDatabaseName = fmt.Errorf("no database name")
-	ErrNilConfig      = fmt.Errorf("no config")
+	ErrNoDatabaseName        = fmt.Errorf("no database name")
+	ErrNilConfig             = fmt.Errorf("no config")
+	ErrTypoAndNotNonTypoUsed = fmt.Errorf("both x-advisory-lock-timeout-interval and x-advisory-lock-timout-interval were specified")
```

Comment:
Since this is an exported specific error, let's make the name specific as well. e.g. something along the lines of: `ErrLockTimeoutConfigConflict`

Reply:
Done